pkgs/applications/audio/plugin-torture/default.nix

```diff
@@ -1,21 +1,22 @@
-{ stdenv, fetchgit, boost, ladspaH, lilv, lv2, pkgconfig, serd, sord, sratom }:
+{ stdenv, fetchFromGitHub, boost, ladspaH, lilv, lv2, pkgconfig, serd, sord, sratom }:
 
 stdenv.mkDerivation rec {
-  name = "plugin-torture-git-${version}";
-  version = "2013-10-03";
+  name = "plugin-torture-v${version}";
```

Comment:
@rasendubi The upstream release name has a v.  Removed it anyway.

@matthewbauer , @rasendubi : Let me know when we're ready for squashing!